nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/utils/ResourceUtil.java

```diff
@@ -248,15 +251,19 @@ public static RestActionReporter runCommand(String commandName,
                                                 boolean managedJob) {            
         AsadminRecorderService asadminRecorderService = Globals.
```

Comment:
`getDefaultHabitat.getService()` can be replaced by the convenience method `get()`